pkg/products/amqonline/reconciler.go

```diff
@@ -412,6 +419,40 @@ func (r *Reconciler) reconcileAddressSpacePlans(ctx context.Context, serverClien
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+func (r *Reconciler) reconcileServiceAdmin(ctx context.Context, serverClient k8sclient.Client, serviceAdminRole *rbacv1.Role) (integreatlyv1alpha1.StatusPhase, error) {
+	r.logger.Info("reconciling service admin role to the dedicated admins group")
+
+	err := serverClient.Create(ctx, serviceAdminRole)
```

Comment:
I generally prefer to see `controllerutil.CreateOrUpdate` where possible. Is there some reason that's not appropriate in this case?

Reply:
I had left it as create only because other resources in the amq online reconciler were create. you are correct, this should be update also. I will update the branch now.

Reply:
Resolved now, thanks @grdryn